src/slider/styled-components.js

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+import {StyledBarProgress} from '../progress-bar';
+
+export const Root = styled('div', props => {
+  return {
+    paddingTop: '1px',
+    position: 'relative',
+  };
+});
+
+export const Axis = styled('div', props => {
+  const {$theme} = props;
+  const {colors} = $theme;
+  return {
+    position: 'relative',
+    margin: '34px 10px 10px',
+    borderRadius: $theme.borders.useRoundedCorners ? '2px' : '0px',
```

Comment:
Should we use values from `$theme` for the border radius px value?

Reply:
ok